packages/docs/Examples/swap.md

```diff
@@ -52,24 +94,16 @@ await bob.sign(tx)
 await bob.broadcast(tx)
 ```
 
-An expression containing both the code of the `Swap` class and the expression `Swap.exec(nftA, nftB)` is passed to the [`encode`](./API/encode.md) function. The second argument is an environment that determines that the smart objects corresponding to the names `nftA` and `nftB` are stored at revisions `nftA._rev` and `nftB._rev`. 
-
-The return value is a transaction `tx` with two inputs and two outputs. The inputs spend the outputs `nftA._rev` and `nftB._rev` representing the two NFTs that are arguments to the function. The two outputs represent the new state of the NFTs after the function call.
-
-The `encode` function will automatically sign all inputs of the transaction that can be signed with the private key of the computer object on which the function is called. In this case, this is the input as revision `nftA._rev`.
-
-Subsequently Bob signs the input `nftB._rev` and broadcasts the transaction. When the transaction is included in the blockchain the swap is executed and the owners of the two NFTs are reversed.
-
 #### Reducing Fees
 
-The disadvantage of the code above is that the swap class is written into the blockchain on every swap. This wasts block space and is expensive. A more efficient approach is to deploy the `Swap` function as a module first and then refer to the module from the transactions executing the swap. To make this easier, we provide a helper class `SwapHelper` for swaps and `TBC721` for NFTs that can be used as follows:
+The disadvantage of the code above is that the swap class is written into the blockchain on every swap. This wasts block space and is expensive. A more efficient approach is to deploy the `Swap` function as a module first and then refer to the module from the transactions executing the swap. To make this easier, we provide a helper class [`SwapHelper`](../../swap/src/swap.ts) for swaps and `TBC721` for NFTs that can be used as follows:
```

Comment:
Will this reference work? I think we need the full path: 
https://github.com/bitcoin-computer/monorepo/blob/main/packages/swap/src/swap.ts